src/routes/transactions/__tests__/resources/incoming-transfers/e2e/erc721-expected-response.json

```diff
@@ -58,7 +58,7 @@
             "tokenId": "110380171670080793190909999014722746036502744273864122623133899039475607835745",
             "tokenName": "Ethereum Name Service",
             "tokenSymbol": "ENS",
-            "logoUri": "/media/tokens/logos/ENS.png"
+            "logoUri": "/media/tokens/logos/0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85.png"
```

Comment:
See https://github.com/5afe/safe-client-gateway-nest/pull/218

If this was done to fix an unrelated E2E test it'd be better to have it in a separate PR.

Reply:
Yes, after rebasing there's no diff anymore, thanks! 👍🏻